README.md

```diff
@@ -31,7 +31,7 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
-$ cmake ..
+$ cmake -DCMAKE_PREFIX_PATH=/opt/ros/indigo ..
```

Comment:
I agree with this assessment, running the workspace setup script sets many environment variables that may be critical to successful builds, including default CMake build modes, for example.

Note that this _cannot_ just be assumed to be `/opt/ros/indigo/setup.bash` because people might have different versions of dependent libraries in their ROS workspace, and this assumption would put them in a state where the aikido build is linked against libraries they are not expecting.

In any case, it seems like either the ROS workspace's full `setup.bash` should to be sourced ahead of time, or none of the ROS variables should be set.  I think this change should be removed, and an optional line added which is something like `source ROS workspace e.g. /opt/ros/indigo/setup.bash` as one of the first steps.

Is Aikido just always dependent on ROS now?  Why is this not being split into a new section like "Compiling for a ROS workspace"?